step-api/src/main/java/org/jenkinsci/plugins/workflow/util/StaplerReferer.java

```diff
@@ -38,8 +38,9 @@
 
 /**
  * JENKINS-19413 workaround.
+ * Will be deleted in the future.
  */
-class StaplerReferer {
+public class StaplerReferer {
```

Comment:
<code>@Restricted</code> or <code>@Deprecated</code> would be useful if you're going to delete it at some point